README.md

```diff
@@ -115,8 +115,6 @@ This is just a simple project under the supervision of the Microverse platform t
 
```

Comment:
![image](https://github.com/tamoorsaeed22/Leaderboard/assets/93039359/a7de1f7e-8a13-4000-b693-1844f1501780)

- [ ] It would be nice if you fix your License link to make your project licensed and your readme file descriptive and professional 😊

## Error 🔴
![image](https://github.com/tamoorsaeed22/Leaderboard/assets/93039359/a9369f2f-15c6-486d-b646-00c5499214f0)

Reply:
@HAMMAS-SALEEM MIT LICENSE key has been added and fix the issue